openapi.yaml

```diff
@@ -5,25 +5,126 @@
 components:
```

Comment:
Pretty weird that there are so many changes but I checked and this is indeed how openapi.yaml is generated on this branch